defcon/plugins/endpoint.py

```diff
@@ -0,0 +1,102 @@
+"""DefCon Prometheus Alertmanager plugin."""
+import requests
+import logging
+
+from defcon.plugins import base
+
+
+class EndpointPlugin(base.Plugin):
+    """DefCon Endpoint plugin.
+
+    set an endpoint to get a defcon level.
+    The endpoint should have the info {...'defcon':[1-5],...}
+    Another endPoint can be setup to force the level
+
+    Config:
+    ```python
+    {
+        'title': 'Defcon Production',
+        'url': 'http://defcon.com/api/simple/production/', // Url to endpoint
+        'link_status': 'http://defcon.com/status/production/ // Url to endpoint info
```

Comment:
Maybe just add `url` and the rest can be guessed from the response

Reply:
done, only url